api/handlers/app.go

```diff
@@ -606,6 +607,32 @@ func (h *App) getSSHEnabled(r *http.Request) (*routing.Response, error) {
 	}), nil
 }
 
+func (h *App) getAppFeature(r *http.Request) (*routing.Response, error) {
+	featureName := routing.URLParam(r, "name")
+	switch featureName {
+	case "ssh":
+		return routing.NewResponse(http.StatusOK).WithBody(map[string]any{
+			"name":        "ssh",
+			"description": "Enable SSHing into the app.",
+			"enabled":     false,
+		}), nil
+	case "revisions":
+		return routing.NewResponse(http.StatusOK).WithBody(map[string]any{
+			"name":        "revisions",
+			"description": "Enable versioning of an application",
+			"enabled":     false,
+		}), nil
+	default:
+		return routing.NewResponse(http.StatusNotFound).WithBody(map[string]any{
```

Comment:
final nitpick:

Instead of returning a response containing an error, the handler here could just return the [special api not-found error](https://github.com/cloudfoundry/korifi/blob/3820a5a049d4b6169e0e29ac896b913626053ea8/api/errors/errors.go#L157) that is going to be presented in a generic manner to the user . Then the switch could be refactored into

```
if featureName == "ssh" {
  return ....
}

if featureName == "revisions" {
  return ...
}

return apierrors.NewNotFoundError(nil, "Feature")
```

Reply:
adapted accordingly